### PR TITLE
redesign settings modal to align button and give a more professional look

### PR DESCRIPTION
**Pull request by AvinaashGhansam**

### CODE REVIEW

**Review by ajd6735 — APPROVED** [review]

File: src/settings/SettingsModal.tsx [hunk 1]
```diff
@@ -62,37 +62,46 @@ const SettingsModal = ({
...
+                  Enable Augmented Reality
                 </FormLabel>
                 <Switch
                   id="ar-toggle"
                   isChecked={arEnabled}
-                  onChange={onChangeArToggle}
+                  onChange={(e) => {
+                    onChangeArToggle(e.target.checked);
+                  }}
+                  style={{ marginLeft: 'auto' }}
```
Comment (zlix12) [inline src/settings/SettingsModal.tsx:77]: Can we set this with a Chakra prop rather than inline style?

File: src/settings/SettingsModal.tsx [hunk 2]
```diff
@@ -62,37 +62,46 @@ const SettingsModal = ({
...
                 <FormLabel htmlFor="ar-toggle" mb="0">
-                  Enable Augmented Reality?
+                  Enable Augmented Reality
                 </FormLabel>
                 <Switch
                   id="ar-toggle"
                   isChecked={arEnabled}
-                  onChange={onChangeArToggle}
+                  onChange={(e) => {
+                    onChangeArToggle(e.target.checked);
```
Comment (zlix12) [inline src/settings/SettingsModal.tsx:75]: Not sure I understand the purpose of this change. Generally want to avoid creating methods in the render loop.

**Review by zlix12 — CHANGES_REQUESTED** [review]
Visually looks great, few comments on implementation

File: src/settings/SettingsModal.tsx (hunk 1 above)
Comment (AvinaashGhansam) [reply src/settings/SettingsModal.tsx:77]: Fixed!

File: src/settings/SettingsModal.tsx (hunk 2 above)
Comment (AvinaashGhansam) [reply src/settings/SettingsModal.tsx:75]: undo these changes

**Review by zlix12 — APPROVED** [review]
LGTM

**Pull request merged**